src/ngraph/frontend/onnx_import/op/clip.cpp

```diff
@@ -43,6 +49,43 @@ namespace ngraph
 
             } // namespace set_1
 
+            namespace set_11
+            {
+                NodeVector clip(const Node& node)
+                {
+                    NodeVector inputs{node.get_ng_inputs()};
+                    std::shared_ptr<ngraph::Node> data = inputs.at(0);
+                    const ngraph::element::Type data_type = data->get_element_type();
+                    const ngraph::Shape data_shape = data->get_shape();
+                    const double max_value = std::numeric_limits<double>::max();
+                    const double min_value = std::numeric_limits<double>::lowest();
+                    std::shared_ptr<ngraph::Node> min =
+                        builder::make_constant(data_type, data_shape, min_value);
+                    std::shared_ptr<ngraph::Node> max =
+                        builder::make_constant(data_type, data_shape, max_value);
+
+                    if (inputs.size() > 1)
+                    {
+                        if (!inputs.at(1)->is_null())
+                        {
+                            min = ngraph::builder::reshape(inputs.at(1), data_shape);
```

Comment:
nGraph's `Minimum` and `Maximum` have support for `auto_broadcast`. Please use that instead of reshape.

Reply:
Done